### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by robaru** [bingo-todd/GCC-PHAT_DNN_Loc#1]
Hi @bingo-todd, 
it is quite a while that I am following your work. 

It has been super useful for me and I can only thank you!

Out of curiosity... are you a involved into academia or these projects are just for fun? :) 

Roberto

**Comment by bingo-todd** [comment]
I am studying for my doctorate in speech processing, and most of my projects are programs I used in my research works.

**Comment by robaru** [comment]
Oh nice,
can I ask you some articles of yours? 

I am also doing a PhD and I am focusing on auditory models, it could be nice to have the chance to cite your work since I am using it! :)

**Comment by bingo-todd** [comment]
yes, I can send it to you by email (please leave your address)  
I only have a few papers, , :rofl:, and all of them focus on binaural sound localization.

**Comment by robaru** [comment]
Super, it is what my PhD is about: binaural sound localization...

My email is barumerli@dei.unipd.it

Thanks

**Comment by bingo-todd** [comment]
Already sent
